salt/docs/init.sls

```diff
@@ -67,29 +67,17 @@ virtualenv-dependencies:
     - onchanges:
       - git: docsbuild-scripts
 
-docsbuild-analytics:
```

Comment:
These cron.env_present will have to be removed manually after https://github.com/python/docsbuild-scripts/pull/269 is live.